include/allegro5/monitor.h

```diff
@@ -16,6 +16,7 @@ typedef struct ALLEGRO_MONITOR_INFO
    int y1;
    int x2;
    int y2;
+   int refresh_rate;
```

Comment:
Sadly we can't do this as this breaks backwards compatibility. It'd need to be a new function `al_get_monitor_refresh_rate`.

Also, if we're using it just in the windows backend (and I see you've only implemented it on Windows), it should be a private function for now.

Reply:
monitor refresh rate can now be retrieved with al_get_monitor_refresh_rate